src/features/foods/components/DeleteFoodDialogContent.tsx

```diff
@@ -63,9 +66,10 @@ export const DeleteFoodDialogContent = ({
       alert('Something went wrong. Please try again.');
     } else {
       alert('Successfully deleted!');
+      onDialogClose();
       onParentClose?.();
+      router.refresh();
     }
-    onDialogClose();
```

Comment:
You probably also need to close the dialog and parent when the request fails.

Reply:
I modified to close dialog and parent also when the request fails!

Do you think this behaviour is friendly to user in the first place?
If the delete request fails, users have to start from opening the parent component and then open the dialog again. Is it maybe better to keep the dialog and parent component open?

Please tell me your point of view! I'm open to change all of the other delete dialogs if you feel so!

Reply:
> Do you think this behaviour is friendly to user in the first place?
If the delete request fails, users have to start from opening the parent comp

Personally, I don't care about the differences. What matters to me and the project is that this dialog behaves differently from the others. It's up to you to decide whether to keep it as it is or change it.

Reply:
Then I'll go for current implementation for now! Please take a look at the latest commit when you have a moment 🙏